internal/engine/upper_test.go

```diff
@@ -1900,6 +1900,26 @@ func TestDockerComposeDetectsCrashes(t *testing.T) {
 	})
 }
 
+func TestDockerComposeBuildCompletedSetsStatusToUp(t *testing.T) {
+	f := newTestFixture(t)
+	m1, _ := f.setupDCFixture()
+
+	expected := container.ID("aaaaaa")
```

Comment:
...do we care that it DOESN'T set status if it doesn't get a containerID back? idk if it's worth a test, but maybe...?